gcp/README.md

```diff
@@ -673,3 +673,55 @@ resolved/update is done.
 If this downtime will be disruptive to your work ("We're giving a demo for the
 Prime Minister at that time!"), please let me know immediately.
 ```
+
+#### External backups
+
+##### Setup.
+
+Create a Terragrunt definition like `gcp/live/prd/k8s/kube-system/backup-exporter/terraform.tfvars` inside the exekube project. This file contains 3 variables:
+   * `destination_bucket` - The destination GCS bucket, i.e "gs://gpii-backup-external-prd". 
+   * `replica_count` - the number of CouchDB replicas that the cluster has. This is important for copying all the snapshots of the cluster at the same time.
+   * `schedule` - Follows the same format as a Cron Job. i.e: `*/10 * * * *` to execute the task every 10 minutes.
+
+i.e:
+```
+# ↓ Module metadata
+
+terragrunt = {
+  terraform {
+    source = "/project/modules//backup-exporter"
+  }
+
+  dependencies {
+    paths = [
+      "../k8s-snapshots",
+    ]
+  }
+
+  include = {
+    path = "${find_in_parent_folders()}"
+  }
+}
+
+# ↓ Module configuration (empty means all default)
+
+destination_bucket = "gs://gpii-backup-dev-alfredo"
+replica_count      = 2
+schedule           = "*/30 * * * *"
+```
+
+##### Sending backup outside of the organization.
+
+The process is mostly executed by the `gcloud compute images export` command, which uses Cloud Build service, which uses [Daisy](ttps://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/daisy) to create a VM, attach the images based on the snapshots to backup, create a file from those images and send the result files to the external storage.
+
+The destination bucket must be created manually. This is a once time setup for STG and PRD, because it doesn't need to be recreated in the future. The bucket must have the following permissions for the service account `[Source_project_number_id]@cloudbuild.gserviceaccount.com` needs the `Storage Admin` roles, because the Cloud Build service needs to copy the final images in the GCS bucket. Note that such service account must exist in order to attach the role. The Cloud Build service account is created when the Cloud Build API is enabled.
```

Comment:
I don't see this documented in the one-time setup doc.

Reply:
In order to make the process more straight forward I added a new Terraform module in the common part that automates the set up of the destination buckets https://github.com/gpii-ops/gpii-infra/pull/400/commits/7befb26d87b521b82864136f095f037fb032c8c8 (PR #400). This doc section is updated in https://github.com/gpii-ops/gpii-infra/pull/432/commits/d8b3176d7c88cc554c2da595e5203e9a602c7211.